sql-statements/sql-statement-alter-table.md

```diff
@@ -147,8 +147,11 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-* 单条 `ALTER TABLE` 语句不能完成多项操作。
-* 当前不支持有损更改，例如从 `BIGINT` 类型更改为 `INT` 类型。
+* 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
+* 不支持主键列上 [Reorg-Data](/sql-statements/sql-statement-modify-column.md#Reorg-Data Change) 类型的变更。
```

Comment:
Follow-uP: anchor error

Reply:
@Liuxiaozhen12 Would you please help modify the anchor in L151 of this doc and in L156 of sql-statements/sql-statement-change-column.md? Thanks!🌹